### DISCUSSION TITLE
How to add 192.168.*.* in  custom rules

### DISCUSSION THREAD

**Discussion by iamsad3508** [shadowsocks/shadowsocks-android#2894] [Q&A]
I want to bypass only 192.168.*.* with route set as ALL, but i know zero about rules, can anyone help?

**Comment by dev4u** [comment]
创建acl文件
```
[proxy_all]
[bypass_list]
192.168.0.0/16
```
然后在app端通过http链接的形式加载

**Reply by iamsad3508** [reply]
先谢谢，按照你的设置好自定义规则，然后在路由里面选择自定义规则。不过，所有的流量似乎都不经过 SS了，IP 没变，192.168..可以访问，测试连接是成功的，而且还没有任何流量统计。有可能是哪里出问题了吗？BUG 吗？
